doc/ansible/dev/dependent_watches.md

```diff
@@ -30,4 +30,117 @@ This is an example of a watches file with the `watchDependentResources` field se
   manageStatus: False
   watchDependentResources: True
 
-```
\ No newline at end of file
+```
+
+### Make existing resources dependent
+
+It is strongly recommended to use dependent watches, resource dependency
+should not be an issue for normal operations. However, if your operator
+deployed resources while [dependent watches were
+disabled](https://github.com/operator-framework/operator-sdk/blob/master/doc/ansible/dev/advanced_options.md#turning-off-dependent-watches-and-owner-reference-injection)
+, this section will show you how to fix it manually (using memcached as
+ the example).
+
+#### Owner Information
+
+First, we retrieve the information that we will later add to the
+dependent resources.
+
+`$ kubectl get memcacheds.cache.example.com -o yaml`
+
+```yaml
+- apiVersion: cache.example.com/v1alpha1
+  kind: Memcached
+  metadata:
+    ...(snip)
+    name: example-memcached
+    uid: ad834522-d9a5-4841-beac-991ff3798c00
+```
+
+#### Namespaced Resources
+
+For namespaced resources, we add an `ownerReference`.
+
+`$ kubectl edit deployment example-memcached-memcached`
+
+```yaml
+
+metadata:
+  ...(snip)
+  ownerReferences:
+    - apiVersion: cache.example.com/v1alpha1
+      kind: Memcached
+      name: example-memcached
+      uid: ad834522-d9a5-4841-beac-991ff3798c00
+```
+
+#### Cluster-scoped Resources
+
+For resources that are not namespaced, `ownerReferences` don't apply, so
+we need to add an `annotation`: (This namespace does not actually exist in the
+memcached example.)
+
+`$ kubectl edit namespace example-memcached`
+
+```yaml
+metadata:
+  annotations:
+    operator-sdk/primary-resource: default/example-memcached
+    operator-sdk/primary-resource-type: Memcached.cache.example.com
+```
+
+Warning: The value for primary-resource-type is "<kind>.<group>". Group
+must be separated from the `apiVersion` client-side. **If the `group` is a
+legacy string without a "/", the script below will fail.**
+
+#### Migration Script
+
+If you have many resources to update, you can modify the following
+**unsupported** script::
+
+```bash
+#!/usr/bin/env bash
+
+# This script is an unsupported example of how to migrate resources that
+# do not # have ownerReferences (or annotations for cluster-scoped
+# resources). Use at your own risk!  
+# https://github.com/operator-framework/operator-sdk/issues/1977
+
+# This script uses jq to parse JSON. https://stedolan.github.io/jq/
+
+# FIXME: Set this to an array of selfLinks of all resources to patch.
+declare -a resources_to_own=("/apis/apps/v1/namespaces/default/deployments/example-memcached-memcached" "/api/v1/namespaces/example-memcached")
+# FIXME: Set this to the name of the CR that should own the resources.
+local owning_cr_name="example-memcached"
```

Comment:
Also maybe allow all of these to be set via environment variables, so that it's easier for a user to automate it if they have a large number of CRs

Reply:
Also I'd add the namespace as a variable, you can pass it in to kubectl with `-n <namespace>` and that way the user won't have to modify their context if they have CRs in more than one namespace.